src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx

```diff
@@ -45,54 +46,79 @@ If your application is [instrumented with <InlinePopover type="browser"/>](/docs
 
     If you see entries in the deny list that are unfamiliar or causing AJAX calls to be filtered, review the [Filter AjaxRequest Events](/docs/browser/new-relic-browser/configuration/filter-ajax-request-events/) documentation for further guidance.
-  </Collapser>
+  </Step>
+
+  <Step>
+
+    ## Verify network access
 
-  <Collapser
-    id="verify-network-access"
-    title="3. Verify network access."
-  >
     If the object is properly instrumented, try triggering an AJAX call in your application while monitoring network traffic in the browser's developer tools. Wait up to one minute, and look for a call to `bam.nr-data.net/jserrors` with an `xhr` parameter. If the call fails, check for network issues.
 
     If you don't see this call, if it fails with an error not related to network access, or if it succeeds but you still aren't seeing data, continue through the troubleshooting documentation.
-  </Collapser>
-</CollapserGroup>
 
-If your requests use JSONP, see requirements and notes on functionality below:
+  </Step>
+
+  <Step>
 
-<CollapserGroup>
-  <Collapser
-    id="jsonp"
-    title="JSONP requirements"
-  >
-    If your requests use [JSONP](https://en.wikipedia.org/wiki/JSONP), these requests will not appear on the [AJAX UI page](/docs/browser/new-relic-browser/browser-pro-features/ajax-page-identifying-time-consuming-calls). However, you can view them as assets within [session traces](/docs/browser/new-relic-browser/browser-pro-features/session-traces-exploring-webpages-life-cycle). If using SPA monitoring, you can view them on the <DNT>**Breakdown**</DNT> tab of the <DNT>**Page views**</DNT> page.
+  If your requests use JSONP, see requirements and notes on functionality below:
+
+  ## JSONP requirements
+  
+    If your requests use [JSONP](https://en.wikipedia.org/wiki/JSONP), these requests will not appear on the [AJAX UI page](/docs/browser/new-relic-browser/browser-pro-features/ajax-page-identifying-time-consuming-calls). 
+    
+    However, you can view them as assets within [session traces](/docs/browser/new-relic-browser/browser-pro-features/session-traces-exploring-webpages-life-cycle). 
+    
+    If using SPA monitoring, you can view them on the <DNT>**Breakdown**</DNT> tab of the <DNT>**Page views**</DNT> page.
 
     Requirements for JSONP to be recognized:
 
     * Each JSONP request must use a unique callback function. Most popular libraries (like jQuery) generate a unique callback function dynamically for each request.
     * The query string containing the callback function name must be named `"callback"` or `"cb"` in order to be recognized by New Relic. This is the default behavior in most popular libraries.
-  </Collapser>
-</CollapserGroup>
 
-If any of these troubleshooting steps fails or you are still having issues with missing AJAX data, get support at [support.newrelic.com](https://support.newrelic.com).
+  <Callout variant="tip">
```

Comment:
@adutta-newrelic since this callout seems to apply to all the options in this troubleshooting guide, I'd move this to the left margin (not have it indented under the bulleted list items.

Reply:
Updated